testing/src/main/java/io/grpc/testing/DeadlineSubject.java

```diff
@@ -92,7 +92,7 @@ public void of(Deadline expected) {
    * A partially specified proposition about an approximate relationship to a {@code deadline}
    * subject using a tolerance.
    */
-  public abstract class TolerantDeadlineComparison {
+  public abstract static class TolerantDeadlineComparison {
 
```

Comment:
I'm not trying to resolve FindBugs notices in this PR. And given that the equals and hashCode throw an exception unconditionally, it seems that it isn't too much of a concern.

The only reason I added static here is that if I don't the build will fail because of the errorprone checks.